ee/tabby-webserver/docs/api_spec.md

```diff
@@ -191,7 +191,7 @@ Redirect to oauth provider for signin
 
 ```shell
```

Comment:
can we merge the example of github / google? they seems to be quite similar to me.